.github/workflows/sentry_delayed_job_test.yml

```diff
@@ -55,7 +55,6 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby_version }}
-          bundler-cache: true
```

Comment:
For some reason, the next step always require `bundle install` for resolving git-based gems. If we were to keep it, then this is not necessary anymore.